src/Microsoft.SqlTools.ServiceLayer/Security/Contracts/LoginRequest.cs

```diff
@@ -39,18 +29,18 @@ public class CreateLoginRequest
         /// Request definition
         /// </summary>
         public static readonly
-            RequestType<CreateLoginParams, CreateLoginResult> Type =
-            RequestType<CreateLoginParams, CreateLoginResult>.Create("security/createlogin");
+            RequestType<CreateLoginParams, object> Type =
```

Comment:
we should probably use a specific type for the return value, just for readability of code.